firewalls.tf

```diff
@@ -1,4 +1,4 @@
-resource "google_compute_firewall" "allow_portrait_engine_metastore_egress" {
+resource "google_compute_firewall" "allow_metastore_egress" {
```

Comment:
**Note:** Given those resources aren't provisioned in LiveRamp's hosted tenant-data project, it won't need to recreate resources (that's good), so it just updates resource naming when remote customer going to use them.